justfile

```diff
@@ -16,7 +16,7 @@ test ARGS="":
     . scripts/setup_functions
     dev_setup
 
-    pytest --cov=. {{ ARGS }}
+    python -m pytest --cov=. --cov-report term-missing {{ ARGS }}
```

Comment:
If we're going to set up reporting by default can we make it this:

    --cov-report=html --cov-report=term-missing:skip-covered

Breaking that down since the docs are kind of spread across both coverage and pytest-cov and I have to look it up every time:
* `html`: generate an HTML report in `htmlcov/index.html` (great visual resource, useful for seeing branch coverage too, might need adding to .gitignore)
* `term-missing`: list files with missing lines and the line ranges where coverage is missing
* `skip-covered`: only show files with missing lines

Reply:
done